tests/reader.test.js

```diff
@@ -0,0 +1,161 @@
+const { expect } = require('chai')
+const request = require('supertest')
+const { Reader } = require('../src/models')
+const app = require('../src/app')
+
+describe('/readers', () => {
+  before(async () => Reader.sequelize.sync())
+
+  beforeEach(async () => {
+    await Reader.destroy({ where: {} })
+  })
+
+  describe('with no records in the database', () => {
+    describe('POST /readers', () => {
+      it('creates a new reader in the database', async () => {
+        const response = await request(app).post('/readers').send({
+          name: 'Elizabeth Bennet',
+          email: 'future_ms_darcy@gmail.com',
+          password: 'testpassword123'
+        })
+        const newReaderRecord = await Reader.findByPk(response.body.id, {
+          raw: true
+        })
+
+        expect(response.status).to.equal(201)
+        expect(response.body.name).to.equal('Elizabeth Bennet')
+        expect(newReaderRecord.name).to.equal('Elizabeth Bennet')
+        expect(response.body.password).to.equal(undefined)
+        expect(newReaderRecord.email).to.equal('future_ms_darcy@gmail.com')
+      })
+    })
+    describe('>----EMAIL Test-----<', () => {
```

Comment:
This should be a more descriptive name really, if you want to create a divider to make the formatting a bit easier to read another describe block would be a better approach. Same applies for the PASSWORD Test on line 44